test/common/stats/isolated_store_impl_test.cc

```diff
@@ -111,6 +112,14 @@ TEST_F(StatsIsolatedStoreImplTest, AllWithSymbolTable) {
   EXPECT_EQ(2UL, store_.gauges().size());
 }
 
+TEST_F(StatsIsolatedStoreImplTest, ConstSymtabAccessor) {
+  ScopePtr scope = store_.createScope("scope.");
+  const Scope& cscope = *scope;
+  const SymbolTable& csymtab = cscope.constSymbolTable();
```

Comment:
Minor: symtab/csymtab didn't immediately mean "symbol table" in my mind. Maybe c_symbol_table?